pkg/cedar/intake/translation/system_intake.go

```diff
@@ -67,7 +67,7 @@ func (si *TranslatableSystemIntake) CreateIntakeModel() (*wire.IntakeInput, erro
 		CostIncreaseAmount:          si.CostIncreaseAmount.Ptr(),
 		Contractor:                  si.Contractor.Ptr(),
 		ContractVehicle:             si.ContractVehicle.Ptr(),
-		ContractNumber:              si.ContractNumber.Ptr(),
+		ContractNumber:              helpers.PointerTo(""), // TODO: populate with linked contract numbers?
```

Comment:
if we want to populate this, we'll need access to the dataloaders here. I tried some different ways of getting the dataloaders in this area, but i was not successful in that.

the alternative is to have a version of the getter that not a loader and use that